tests/data/validations/virtualservices/non-existent-gateway-in-match.yaml

```diff
@@ -0,0 +1,34 @@
+kind: VirtualService
+apiVersion: networking.istio.io/v1alpha3
+metadata:
+  name: test
+  namespace: default
+spec:
+  hosts:
+    - '*'
+  gateways:
+    - valid-gateway
+  http:
+    - match:
+        - gateways:
+            - valid-gateway
+        - uri:
+            exact: /static        
+      route:
+        - destination:
+            host: test
+            port:
+              number: 9080
+    - match:
+        - gateways:
+            - valid-gateway
+            - non-existent-bookinfo-gateway        
```

Comment:
What do you think of adding the same validation with the long nomenclature for gateways `ns/gw`? This is to make sure the validation will appear using all the available formats.

Reply:
I agree, I added to the test scenarios, let me know what do you think, thanks!.